packages/python/src/components/Method.tsx

```diff
@@ -0,0 +1,57 @@
+import { Children, Indent, Scope } from "@alloy-js/core";
+import { usePythonNamePolicy } from "../name-policy.js";
+import { Declaration } from "./Declaration.jsx";
+import { Parameters, ParametersProps } from "./Parameters.jsx";
+
+export interface MethodProps extends ParametersProps {
+  name: string; // e.g. "__init__" or "foo"
+  isInstanceMethod?: boolean; // true if this is an instance method
```

Comment:
I would call these `instance` and `class`, or `instanceMethod` and `classMethod`. Is prefix typically isn't used in the context of JSX component boolean props.

Reply:
Fixed.